### PR TITLE
refactor: move `@pnpm/logger` to the monorepo

### PR DESCRIPTION
**Pull request by KSXGitHub**

### CODE REVIEW

File: workspace/find-packages/package.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
     "@pnpm/workspace.read-manifest": "workspace:*"
   },
   "peerDependencies": {
-    "@pnpm/logger": "^5.0.0"
+    "@pnpm/logger": "workspace:*"
```
Comment (zkochan) [inline workspace/find-packages/package.json]: keep the peer deps unchanged and add it to dev deps

File: .changeset/fast-comics-flow.md
Comment (zkochan) [inline .changeset/fast-comics-flow.md:1]: no need to release anything

**Comment by KSXGitHub** [comment]
I used a [script](https://gist.github.com/KSXGitHub/9c6e1bf272025b235380f23dc21828ad) to update the `package.json` file, so some unordered `devDependencies` keys may have been sorted.

**Comment by zkochan** [comment]
You didn't have to create a new script as one is already present: https://github.com/pnpm/pnpm/blob/main/.meta-updater/src/index.ts

**Review by zkochan — APPROVED** [review]

**Pull request merged**